shared/karma/shared.karma.conf.js

```diff
@@ -41,9 +41,22 @@ function getConfig(config, env) {
 
   // For backwards compatability, Karma overwrites arrays
   config.reporters.push('junit');
-  config.coverageReporter.reporters.push({
-    type: 'cobertura'
-  });
+
+  // Backwards compatability for coverage reporters
+  // https://github.com/blackbaud/skyux-sdk-builder/commit/319d2ed2e63cd5208490e6816fefecb68850bbe1#diff-51c68ad4ecf9f8de675a11d4ee5a3fe7R75
+  const coberturaReportType = 'cobertura';
+
+  // @skyux-sdk/builder <= 3.5.3
+  if (config.coverageReporter && Array.isArray(config.coverageReporter.reporters)) {
+    config.coverageReporter.reporters.push({
+      type: coberturaReportType
+    });
+  }
+
+  // @skyux-sdk/builder >= 3.6.0
+  if (config.coverageIstanbulReporter && Array.isArray(config.coverageIstanbulReporter.reports)) {
+    config.coverageIstanbulReporter.reports.push(coberturaReportType);
+  }
```

Comment:
Looks pretty straightforward. Just on minor question - Before, there would always be a default type of `cobertura` pushed to the array no matter what. We don't have any `else` blocks here to have a default fallback. Does that concern you?

Reply:
Great question Alex.  That functionality, of always calling `.push()` is actually where the bug is.  In a release of builder, we changed from `coverageReporter.reporters` to `coverageIstanbulReporter.reports`. This means, in it's current state, depending on the version of builder, we're attempting to call `.push()` on `undefined`.  All that's to say, if neither of these exist, that's ok since there's nothing we need to push the reporter to.

Reply:
sounds good. ship it!